File: react-native/ios/Keybase/SwiftTest.m
//  Keybase
//
//  Created by Chris Nojima on 8/25/15.
//  Copyright (c) 2015 Keybase. All rights reserved.
//

#import "RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(SwiftTest, NSObject)

RCT_EXTERN_METHOD(example:(NSString *)prefix callback:(RCTResponseSenderBlock)callback)

@end

File: react-native/ios/Keybase/SwiftTest.swift
//  Keybase
//
//  Created by Chris Nojima on 8/25/15.
//  Copyright (c) 2015 Keybase. All rights reserved.
//

import Foundation

@objc(SwiftTest)
class SwiftTest: NSObject {

  @objc func example(prefix: String, callback: RCTResponseSenderBlock) -> Void {
    callback([prefix + " from Swift"])
  }

}